source/usage-examples/count.txt

```diff
@@ -0,0 +1,61 @@
+=============================================
+Count the Number of Documents in a Collection
+=============================================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+The Node.js driver provides two methods for counting the number of
+documents in a collection:
+
+- :node-api:`collection.countDocuments()
+  </Collection.html#countDocuments>` returns the number of documents in
+  the collection that match the specified query. If you specify a empty
+  query document, ``countDocuments()`` returns the total number of
+  documents in the collection.
+
+- :node-api:`collection.estimatedDocumentCount
+  </Collection.html#estimatedDocumentCount>` returns an
+  **estimation** of the number of documents in the collection based on
+  collection metadata.
+
+``estimatedDocumentCount()`` is faster than ``countDocuments()`` because
+the estimation is based on the collection's metadata rather than scanning
+the collection. In contrast,
+``countDocuments()`` takes longer to return, but provides an
+**accurate** count of the number of documents and supports specifying a
+filter. Choose the appropriate method for your workload.
+
+``countDocuments()`` accepts a :manual:`query
+</tutorial/query-documents>` parameter. ``countDocuments()`` will only
+count the documents that match the specified query.
+
+``countDocuments()`` and ``estimatedDocumentCount()`` support optional
+settings that affect the method's execution. Refer to the reference
+documentation for each method for more information.
+
+``countDocuments()`` and ``estimatedDocumentCount()`` behave in two
```

Comment:
Four paragraphs in a row that begin with a method name instead of an english word *feels* subjectively reference-y to me. Do you think we could spice this up a little bit, maybe with something like 
```
"the ``countDocuments()`` and ``estimatedDocumentCount()`` methods"
```
just to vary it up/make it feel less formal?

Of course there's nothing wrong with it as is, so use your best judgment -- this is a "look and feel" suggestion.

Reply:
Yup, fair! I'm tweaking.